File: js/PROBLEMS/1074.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

e = lines[0]
str = ''
for (i=1;i<=e;i++){
    if (lines[i] == 0){
        console.log('NULL')

    }else {
        if (lines[i]%2==0){
            str+= "EVEN"
        }else{
            str+= "ODD"
        }
    
        if (lines[i] > 0){
            str+=' POSITIVE'
        }else{
            str+=' NEGATIVE'
        }
        console.log(str)
    }

    str=''
}

File: python/PROBLEMS/1074.py
e = int(input())

for i in range(0, e):
    x = int(input())
    str=''
    if (x == 0):
        print("NULL")
    
    else:
        if (x % 2 == 0):
            str+="EVEN" 
        
        else:
            str+="ODD" 
        

        if (x > 0):
            str+=" POSITIVE" 
        
        else:
            str+=" NEGATIVE" 
        
        print(str)
        